src/request.rs

```diff
@@ -20,25 +21,33 @@ pub(crate) async fn request<Input: Serialize + std::fmt::Debug, Output: 'static
 ) -> Result<Output, Error> {
     trace!("{:?} on {}", method, url);
 
-    let client = reqwest::Client::new();
-
-    let response = match &method {
-        Method::Get => client.get(url).header("X-Meili-API-Key", apikey).send().await?,
-        Method::Delete => client.delete(url).header("X-Meili-API-Key", apikey).send().await?,
-        Method::Post(body) => client.post(url)
+    let mut response = match &method {
+        Method::Get => Request::get(url)
+            .header("X-Meili-API-Key", apikey)
+            .body(())
+            .map_err(|e| isahc::Error::InvalidHttpFormat(e))?
```

Comment:
this is a bit ugly, should I add an `impl From<isahc::http::Error> for Error` instead?

Reply:
It only takes one line. It's not useful to implement `From<isahc::http::Error>`.

Reply:
4x1 line, about as much as the `From<isahc::Error>`. But i have no strong opinion here.